src/config.template.cjs

```diff
@@ -321,7 +322,8 @@ module.exports = {
 	 * Whether to inject using the same labels/categories as the original
 	 * torrent.
 	 *
-	 * qBittorrent: This will apply the category's original category as a tag.
+	 * qBittorrent: When using linking, this will apply the category's original
+	 * category as a tag.
```

Comment:
```suggestion
	 * category as a tag when duplicate categories is true.
```
?

Reply:
IIRC the only difference is that with true, .cross-seed is appended to the tag, doesn't false still tag the original category without .cross-seed appended?

Frankly, @ShanaryS should be doubled checking the qBit documentation - as he was the last one to fix up the bullshit tag autotmm category non-sense you guys had requested lol

Either way, I'll fix it to what I think it is, but @ShanaryS will have to give the OK on this, you know since you don't read the docs @bakerboy448

Reply:
Pushed a more concise and thorough description of all the behavior, but needs validation.

Reply:
`duplicateCategories: false` will not apply any extra tags. `duplicateCategories: true` will apply `category.cross-seed`.

Reply:
Ok, well can you push the corrections as you had last had them?